superflore/generators/bitbake/yocto_recipe.py

```diff
@@ -134,18 +136,8 @@ def get_recipe_text(self, distributor, license_text, die_msg=None):
             self.license = self.license.replace(' ', '-')
             ret += 'LICENSE = "' + self.license + '"\n'
```

Comment:
@shaneallcroft You might update this one as well.

Reply:
Oh SNAP i gotchu